llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -411,6 +417,21 @@ std::string saveModuleIR(Module &M, int I, StringRef Suff) {
   return OutFilename;
 }
 
+bool isImportedFunction(const Function &F) {
+  // Functions with definitions are not imported
+  if (!F.isDeclaration())
+    return false;
+
+  bool ReturnValue = true;
+  if (auto NameStr = itaniumDemangle(F.getName())) {
+    StringRef DemangledName(NameStr);
+    if (DemangledName.starts_with("__") || DemangledName.starts_with("_spirv"))
```

Comment:
The prefix for spir-v builtins is __spirv. So, I do not think you need to check for that as you are already checking for __ prefix.

Thanks

Reply:
Changed to only import "__" prefix.